Indicate support for Linux 2.6.32

Although upstream hasn't updated its LATEST_VERSION macro, this version
works without modification on Linux 2.6.32.  This patch squelches a
warning.

Signed-off-by: John Wright <jsw@debian.org>
diff --git a/makedumpfile.h b/makedumpfile.h
index 656096b..9a0ca4d 100644
--- a/makedumpfile.h
+++ b/makedumpfile.h
@@ -449,7 +449,7 @@ do { \
 #define KVER_MIN_SHIFT 16
 #define KERNEL_VERSION(x,y,z) (((x) << KVER_MAJ_SHIFT) | ((y) << KVER_MIN_SHIFT) | (z))
 #define OLDEST_VERSION		KERNEL_VERSION(2, 6, 15)/* linux-2.6.15 */
-#define LATEST_VERSION		KERNEL_VERSION(2, 6, 31)/* linux-2.6.31 */
+#define LATEST_VERSION		KERNEL_VERSION(2, 6, 32)/* linux-2.6.32 */
 
 /*
  * vmcoreinfo in /proc/vmcore
